source/transfer_list.rst

```diff
@@ -141,15 +141,39 @@ The TE header is defined in :numref:`tab_te_header`.
      - The size of this entry header in bytes. This field is set to 8 for the TE header layout described in this version of the table.
 
 
-Entry type ranges
+Entry type allocation
 -----------------
 
-The content of the data section is determined by the tag id. The tag id space contains two ranges:
-
- #. Standard range, and
- #. Non-standard range
-
-The *tag_id* ranges are described in :numref:`tab_tag_id_ranges`.
+Tag IDs must be allocated in this specification before use. A new tag ID can be
+allocated by submitting a pull request to this repository that adds a description
+of the respective TE data layout to this specification. Tag IDs do not have to be
+allocated in order. Submitters are encouraged to try to group tag IDs together in
+logical clusters (e.g. all tags related to a particular chipset or to a
+particular firmware project could use adjacent tag numbers), but there are no
+predefined ranges and no reservations of tag ranges for specific use. The
+initially legal range for tag allocations is {0, ..., 0xf_ffff} (more ranges
+will be opened up when necessary).
+
+Tags are expected to have a simple layout and each tag should only represent
+data for a single logical concept. Data for multiple distinct concepts should be
+split across different tags, even if they're always expected to appear together
+on the first platform adding the tag (to encourage reusability in different
+situations). A list of objects can be represented by adding multiple TEs with
+the same tag ID to the TL.
```

Comment:
Anything like this should really be in the FDT, as it says in the current version.

The TL is just a way to provide things for very early firmware usage, where memory is limited